### ISSUE TITLE
Feature proposal: Render Statement with passed File

### ISSUE THREAD

**Issue by DsXack** [dave/jennifer#55]
Sometimes I need to render only body of File without imports to append exists file. So I need to call GoString() method on Statement and pass File (with presetted import aliases) as param to use valid import aliases in the code.

**Comment by DsXack** [comment]
```go
// RenderWithFile renders the Statement to the provided writer.
func (s *Statement) RenderWithFile(writer io.Writer, f *File) error {
	buf := &bytes.Buffer{}
	if err := s.render(f, buf, nil); err != nil {
		return err
	}
	b, err := format.Source(buf.Bytes())
	if err != nil {
		return fmt.Errorf("Error %s while formatting source:\n%s", err, buf.String())
	}
	if _, err := writer.Write(b); err != nil {
		return err
	}
	return nil
}

// GoStringWithFile renders the Statement for testing. Any error will cause a panic.
func (s *Statement) GoStringWithFile(f *File) string {
	buf := bytes.Buffer{}
	if err := s.RenderWithFile(&buf, f); err != nil {
		panic(err)
	}
	return buf.String()
}

```

**Comment by DsXack** [comment]
Fixed in #56
